students/Gorthi_kavita/6module/6unittest.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import mailtest
+
+class Testmail(unittest.TestCase):
+
+    def test_report(self):
+        mailtest.fmenureport
+
+
+    def test_menu(self):
+       mailtest.menuprint
+
+
+    def test_thankmail(self):
+        mailtest.thankmail
+
+    def test_menuexit(self):
+        mailtest.fmenu3
+
+
+    def test_nodonation(self):
+        mailtest.fask
+        print("test 1 print the  report")
+        print("test 2 print the menu")
+        print("test 3 send thankyou mail")
+        print("test 4 user enter no donation")
+        print("test 5 menu exit")
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
You tests should `assert` (test) what is returned is what you expect, not just that it called a function.

Reply:
https://jeffknupp.com/blog/2013/12/09/improve-your-python-understanding-unit-testing/